src/sentry/demo/demo_org_manager.py

```diff
@@ -48,31 +50,20 @@ def create_demo_org(quick=False) -> Organization:
             )
 
             team = org.team_set.create(name=org.name)
-            python_project = Project.objects.create(
-                name="Python", organization=org, platform="python"
-            )
-            python_project.add_team(team)
 
-            react_project = Project.objects.create(
-                name="React", organization=org, platform="javascript-react"
-            )
-            react_project.add_team(team)
+            def create_project(name, platform):
```

Comment:
@roggenkemper Nit: I would suggest making this function take just `**kwargs` then do: `Project.objects.create(**kwargs)`

Reply:
eh, on second thought it's nice you don't have to pass in the org. what you have is fine as-is